src/OpenTelemetry.Instrumentation.AWSLambda/README.md

```diff
@@ -28,6 +28,19 @@ TracerProvider tracerProvider = Sdk.CreateTracerProviderBuilder()
         .Build();
 ```
 
+Alternatively, you can use `AddAWSLambdaDetector()` on the `ResourceBuilder` to only add
+the AWS Lambda resource detector without the additional Lambda-specific instrumentation:
```

Comment:
```suggestion
the AWS Lambda resource detector without additional instrumentation:
```

"Lambda-specific" also applies to the reource detectector, so it seems odd to call it out for the instrumentation